riva/proto/riva_nmt.proto

```diff
@@ -77,8 +77,12 @@ message SynthesizeSpeechConfig {
 *
 */
 message StreamingTranslateSpeechToSpeechResponse {
-        // Contains speech responses, the last response sends an empty buffer to mark the end of stream.
-        nvidia.riva.tts.SynthesizeSpeechResponse speech = 1; //from riva_tts.proto
+    // Contains speech responses, the last response sends an empty buffer to mark the end of stream.
+    nvidia.riva.tts.SynthesizeSpeechResponse speech = 1; //from riva_tts.proto
+
+    // The ID associated with the request. This is a UUID specific only to
+    // the given request.
+    string request_id = 2;
```

Comment:
thoughts on:

message UUID {
  required string value = 1;
}

or
message request_id {
 required string value = 1;
}

Reply:
Do we need new message type? Will prefer a direct field for simplicity.

Reply:
Keeping it as string allows clients to pass request id in any format other than UUID.

Reply:
Seems ok to me to add it in existing message fields, since that seems more coupled with a particular request/response than having a separate message.

Reply:
ok, I misunderstood. Seems @junkin meant a nested message inside existing message.